packages/easy-testing/src/StaticFixtureSplitter.php

```diff
@@ -39,18 +39,19 @@ public static function splitFileInfoToInputAndExpected(SmartFileInfo $smartFileI
 
     public static function splitFileInfoToLocalInputAndExpectedFileInfos(
         SmartFileInfo $smartFileInfo,
-        bool $autoloadTestFixture = false
+        bool $autoloadTestFixture = false,
+        string $prefix = ''
     ): InputFileInfoAndExpectedFileInfo {
         $inputAndExpected = self::splitFileInfoToInputAndExpected($smartFileInfo);
 
-        $inputFileInfo = self::createTemporaryFileInfo($smartFileInfo, 'input', $inputAndExpected->getInput());
+        $inputFileInfo = self::createTemporaryFileInfo($smartFileInfo, $prefix.'input', $inputAndExpected->getInput());
```

Comment:
@TomasVotruba Except the directory name of the fixture is not being used by the test because as you can see on this line it's moved to a temp directory without preserving the path structure, thus the prefix is needed and just reflects the fixture dir name

Reply:
I see. This is a bit hacky fix and it changes external test package.

I wonder if there is a better solution, e.g. unrelated with file path at all.

Reply:
Well the thruth is symonfy uses the file path to match thoses files, so using something else feels like risky behavior as it's likely not to give the expected result in all cases

Reply:
I added the ability to preserve the directory structure of the fixture instead of hardcoding the value

Reply:
This is amazing work :+1: